numpy/random/_common.pyx

```diff
@@ -220,7 +220,7 @@ cdef np.ndarray int_to_array(object value, object name, object bits, object uint
 
 cdef validate_output_shape(iter_shape, np.ndarray output):
     cdef np.npy_intp *shape
-    cdef ndim, i
+    cdef np.npy_intp ndim, i
     cdef bint error
     dims = np.PyArray_DIMS(output)
```

Comment:
Since we're here, I guess this should be `shape` rather than `dims` (or else fix the `cdef`)

Reply:
Fixed